_el/exercises.md

```diff
@@ -1,169 +1,95 @@
-then looks at the state of the art in automated grading, and closes by
-exploring discussion, projects, and other important kinds of work that
-require more human attention to assess. Our discussion draws in part on
-the [Canterbury Question Bank](http://web-cat.org/questionbank/)
-[[Sand2013](#CITE)], which has entries for various languages and topics
-in introductory computing.
-
-## The Classics {#s:exercises-classics}
-
-As [s:models-formative-assessment](#SECTION) discussed, *multiple
-choice questions* (MCQs) are most effective when the wrong answers probe
-for specific misconceptions. In terms of Bloom's Taxonomy
-([s:process-objectives](#SECTION)), MCQs are usually designed to test
-recall and understanding ("What is the capital of Saskatchewan?"), but
-they can also require learners to exercise judgment.
-
-> **A Multiple Choice Question**
-> 
-> In what order do operations occur when the computer evaluates the
-> expression `price = addTaxes(cost - discount)`?
-> 
-> 1. subtraction, function call, assignment
-> 1. function call, subtraction, assignment
-> 1. function call, then assignment and subtraction simultaneously
-> 1. none of the above
-
-The second classic type of programming exercise is *code and run*
-(C&R), in which the learner writes code that produces a specified
-output. C&R exercises can be as simple or as complex as the teacher
-wants, but for in-class use, they should be brief and have only one or
-two plausible correct answers. For novices, it's often enough to ask
-them to call a specific function: experienced teachers often forget how
-hard it can be to figure out which parameters go where. For more
-advanced learners, figuring out which function to call is more engaging
-and a better gauge of their understanding.
-
-> **Code & Run**
-> 
-> The variable `picture` contains a full-color image read from a file.
-> Using one function, create a black and white version of the image and
-> assign it to a new variable called `monochrome`.
+Κάθε καλός ξυλουργός έχει μια σειρά από κατσαβίδια και κάθε καλός δάσκαλος έχει διαφορετικά είδη ασκήσεων διαμορφωτικής αξιολόγησης τα οποία μπορεί να χρησιμοποιήσει για να ελέγχει τι μαθαίνουν πραγματικά οι μαθητές, να τους βοηθά να εξασκούν τις νέες τους δεξιότητες, και να διατηρεί ζωντανό το ενδιαφέρον τους. Αυτό το κεφάλαιο αρχίζει περιγράφοντας διάφορα είδη ασκήσεων που μπορείτε να χρησιμοποιήσετε για να ελέγξετε αν η διδασκαλία σας ήταν αποτελεσματική. Στη συνέχεια κάνει μια επισκόπηση της τρέχουσας κατάστασης στην τεχνολογία αυτοματοποιημένη αξιολόγησης και ολοκληρώνεται με μια διερεύνηση των συζητήσεων, έργων, και άλλων σημαντικών ειδών εργασίας τα οποία απαιτούν περισσότερη ανθρώπινη προσοχή για να αξιολογηθούν. Η συζήτησή μας βασίζεται εν μέρει στην [Τράπεζα Ερωτήσεων του Canterbury] (http://web-cat.org/questionbank/)[[Sand2013] (# CITE)], η οποία περιέχει καταχωρήσεις για διάφορες γλώσσες προγραμματισμού και θέματα που αφορούν την εισαγωγικού επιπέδου υπολογιστική.
```

Comment:
1. διαμορφωτική αξιολόγηση--> ενδιάμεση αξιολόγηση (βλ. παραπάνω σχόλιο)
2. την τεχνολογία αυτοματοποιημένη αξιολόγησης--> στην τεχνολογία της αυτοματοποιημένης διόρθωσης
3. και ολοκληρώνεται με--> και καταλήγει σε
4. έργων, και άλλων--> των έργων, και των άλλων

Reply:
Thanks @pdelias ! Fixed in a new commit. Εξαίρεση είναι το κομμάτι της "διαμορφωτικής αξιολόγησης" (see previous comment).